### PR TITLE
Intune - Update windows-10-expedite-updates.md

### PR DESCRIPTION
**Pull request by unicornpancakes**
Grammatical changes and various rewordings to make the document more legible.  Does not materially impact the message being communicated, these were just tense and grammatical errors that I think should be corrected.

### CODE REVIEW

**Review by Smritib17 — APPROVED** [review]

File: memdocs/intune/protect/windows-10-expedite-updates.md [hunk 1]
```diff
@@ -31,19 +31,19 @@ ms.collection:
...
 
 ## How expedited updates work
 
-With expedited updates, you can speed installation of quality updates like the most recent *patch Tuesday* release or an out-of-band security update for a zero-day flaw.
+With expedited updates, you can expedite the installation of quality updates like the most recent *patch Tuesday* release or an out-of-band security update for a zero-day flaw.
 
-To speed installation, expedite is able to check for expedited updates more frequently than the normal Windows Update scan frequency. This process enables devices to start the download and install of an expedited update as soon as possible, without having to wait for the device to check in for updates.
+Expedited update policies temporarily override deferrals and other settings to install updates as quickly as possible. This process enables devices to start the download and install of an expedited update without having to wait for the device to check in for updates.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/windows-10-expedite-updates.md:42]:
```suggestion
Expedited update policies temporarily override deferrals and other settings to install updates as quickly as possible. This process enables devices to start the download and installation of an expedited update without having to wait for the device to check in for updates.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @unicornpancakes, kindly commit the additional suggested change. Many thanks.

**Comment by unicornpancakes** [comment]
> Copy review - @unicornpancakes, kindly commit the additional suggested change. Many thanks.

Thanks for the feedback. Commit the change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Smritib17** [comment]
#sign-off

**Pull request merged**